obelix/idefix/producer/producer.go

```diff
@@ -0,0 +1,37 @@
+package producer
+
+import (
+	"fmt"
+	"github.com/Shopify/sarama"
+	"os"
+)
+
+var brokers []string
+
+func init() {
+	brokers = []string{os.Getenv("PRODUCER_ENDPOINT")}
+}
+
+func New() (sarama.SyncProducer, error) {
+	config := sarama.NewConfig()
+	config.Producer.Partitioner = sarama.NewRandomPartitioner
+	config.Producer.RequiredAcks = sarama.WaitForAll
+	config.Producer.Return.Successes = true
+	producer, err := sarama.NewSyncProducer(brokers, config)
+
+	if err != nil {
+		return nil, fmt.Errorf("failed to create sync producer: %v", err)
```

Comment:
Please use %w instead of %v.

Reply:
e45fc0697cd622b70cb923c8230cdba553212bcf